tests/TestRvsdgs.cpp

```diff
@@ -2883,10 +2885,9 @@ MemcpyTest::SetupRvsdg()
   };
 
   auto SetupFunctionG = [&](
-    delta::output & localArray,
-    delta::output & globalArray,
-    lambda::output & lambdaF)
-  {
+    delta::output &localArray,
+    delta::output &globalArray,
+    lambda::output &lambdaF) {
```

Comment:
and here

Reply:
Overhauled the TestRvsdgs.hpp and TestRvsdgs.cpp